service/Dockerfile

```diff
@@ -2,10 +2,12 @@
 FROM openjdk:8-jre
 
 WORKDIR /usr/local/src
-COPY . /usr/local/src
+COPY . /usr/local/src/
+
+RUN rm -rf /usr/local/src/target
 
 ADD target/scala-2.10/geotrellis-transit-assembly-0.1.0-SNAPSHOT.jar /opt/
 
 EXPOSE 9999
 
-CMD java -jar /opt/geotrellis-transit-assembly-0.1.0-SNAPSHOT.jar server /usr/local/src/production.json
+CMD java -Xmx2g -jar /opt/geotrellis-transit-assembly-0.1.0-SNAPSHOT.jar server /usr/local/src/production.json
```

Comment:
For more flexibility, I think that the `ENTRYPOINT` and `CMD` should be configured like:

```dockerfile
ENTRYPOINT ["java"]
CMD ["-jar", "project-name.jar"]
```